app/routes/messages.rb

```diff
@@ -16,15 +16,27 @@ def assignment_query
     with_organization_and_course 'exercise.eid': json_body[:exercise_id], 'student.uid': json_body[:uid]
   end
 
+  def find_or_create_suggestion(assignment)
```

Comment:
Mmm, I don't agree here. `suggestion_id` is a nullable field that can or cannot be included in the POST request. Why should the model be aware of this?